client/components/activity-card/style.scss

```diff
@@ -152,87 +152,6 @@ input[type='text'].form-text-input.activity-card__share-email {
 	margin-bottom: 16px;
 }
 
-.activity-card__activity-actions,
-.activity-card__activity-actions-reverse {
-	align-items: center;
-	border-top: 1px solid var( --color-neutral-5 );
-	display: flex;
-	justify-content: space-between;
-	padding-top: 0.7rem;
-	// balance top and bottom against the 8px overall card padding
-	padding-bottom: calc( 0.7rem - 8px );
-	position: relative;
-	top: 0.5rem;
-
-	a,
-	button.button.is-borderless.is-compact {
-		font-weight: 400;
-		font-size: $font-body-extra-small;
-		display: flex;
-		align-items: center;
-		@include breakpoint-deprecated( '>660px' ) {
-			font-size: $font-body;
-		}
-
-		// gridicons within buttons have very specific rules for font sizes, disable them to let flex take care of it
-		.gridicon {
-			display: block;
-			margin-top: 0;
-			top: 0;
-		}
-	}
-}
-
-.activity-card__activity-actions-reverse {
-	flex-direction: row-reverse;
-}
-
-.backup-delta__realtime .activity-card__actions-button.button.is-borderless.is-primary {
-	float: right;
-}
-
-.button.is-borderless.is-primary:focus {
```

Comment:
Does this rule has the potential to alter styles in the rest of the site?

Reply:
This was my worry as well, which is why I re-wrote this rule in `toolbar/style.scss` to scope it specifically to `.toolbar__button.button`. I do wonder, though, if this might be the underlying cause of the issue you found where buttons are keeping their focused state. 🤔